db/models/index.js

```diff
@@ -37,9 +41,28 @@ const db = {
     Feedback: Feedback.init(sequelize, Sequelize),
     SurveyAnswer: SurveyAnswer.init(sequelize, Sequelize),
     Order: Order.init(sequelize, Sequelize),
-    OrderItem: OrderItem.init(sequelize, Sequelize)
+    OrderItem: OrderItem.init(sequelize, Sequelize),
+    ApplicationEmailQueue: ApplicationEmailQueue.init(sequelize, Sequelize)
 };
 
+// Relations
+db.PendingApplication.belongsTo(db.Users, {
+    constraints: false // don't delete users when deleting applications
+});
+db.Users.hasMany(db.PendingApplication);
+
+db.SubmittedApplication.belongsTo(db.Users);
+db.Users.hasMany(db.SubmittedApplication);
+
+db.ApplicationEmailQueue.belongsTo(db.PendingApplication, {
+    foreignKey: 'applicationId',
+    constraints: false
```

Comment:
Could we use `cascade` to delete emailQueue records when the appropriate pending applications are deleted ?

Reply:
We could but I'm a bit wary of it as it's a bit magic and I'm scared of getting the associations wrong (the docs are fairly unclear about this) and accidentally deleting a user's account as well when we delete email queue items (or pending applications). For now I've just manually deleted them which I think is more explicit.

Reply:
Yeah I agree with the docs being unclear. Makes sense for it to explicit!